action.yml

```diff
@@ -0,0 +1,146 @@
+    description: Name of the GKE Cluster
+    required: true
+  GKEClusterMachineType:
+    description: Name of the machine type used for the GKE Cluster
+    default: 'n1-standard-4'
+    required: false
+  GKEClusterNumNodes:
+    description: Number of nodes for the GKE Cluster
+    default: '3'
+    required: false
+  GCloudComputeZone:
+    description: Name of the GCloud compute Zone
+    default: 'us-east1-b'
+    required: false
+  GCloudServiceKey:
+    description: Base64 encoded GCloud service Key JSON
+    required: true
+outputs:
+  KeptnAPIURL:
+    value: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_ENDPOINT }}
+    description: URL of the created Keptn instance
+  KeptnAPIToken:
+    value: ${{ steps.authenticate_keptn_cli.outputs.KEPTN_API_TOKEN }}
+    description: API Token of the created Keptn instance
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      id: create_downloads_folder
+      run: mkdir ${HOME}/downloads
+    - shell: bash
+      id: create_gke_cluster
+      env:
+        GKE_VERSION: ${{ inputs.GKEClusterVersion }}
+        GKE_CLUSTER_NAME: ${{ inputs.GKEClusterName }}
+        GCLOUD_SERVICE_KEY: ${{ inputs.GCloudServiceKey }}
+        GCLOUD_PROJECT_NAME: ${{ inputs.GKEProjectName }}
+        CLOUDSDK_COMPUTE_ZONE: ${{ inputs.GCloudComputeZone }}
+        GKE_CLUSTER_MACHINE_TYPE: ${{ inputs.GKEClusterMachineType }}
+        GKE_CLUSTER_NUM_NODES: ${{ inputs.GKEClusterNumNodes }}
+        CLOUDSDK_REGION: ${{ inputs.GKEClusterRegion }}
+      run: |
+        echo "Installing gcloud CLI"
+        export OS_TYPE="linux"
+        ${{ github.action_path }}/download_and_install_gcloud.sh
+        echo ${GCLOUD_SERVICE_KEY} | base64 --decode > ~/gcloud-service-key.json
+        gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+        ${{ github.action_path }}/gke_create_cluster.sh
+        ${{ github.action_path }}/gke_authenticate_at_cluster.sh
+
+    - shell: bash
+      id: install_istio
+      run: ${{ github.action_path }}/install_istio.sh
+
+    - shell: bash
+      id: install_keptn_cli
+      env:
+        KEPTN_VERSION: ${{ inputs.KeptnVersion }}
+      run: ${{ github.action_path }}/download_and_install_keptn.sh
+
+    - shell: bash
+      id: install_keptn
+      run: ${{ github.action_path }}/install_keptn.sh
+
+    - shell: bash
+      id: authenticate_keptn_cli
+      run: |
+        source ./utils.sh
+        # authenticate at Keptn API
+        KEPTN_ENDPOINT=http://$(kubectl -n keptn get service api-gateway-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')/api
+
+        KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
+
+        echo "KEPTN_ENDPOINT=${KEPTN_ENDPOINT}"
+
+        auth_at_keptn $KEPTN_ENDPOINT $KEPTN_API_TOKEN
+        verify_test_step $? "Could not authenticate at Keptn API"
+
+        echo "##[set-output name=KEPTN_ENDPOINT;]$(echo ${KEPTN_ENDPOINT})"
```

Comment:
Does this mean that these two lines are visible in the log output?

Reply:
no, they are not